README.md

```diff
@@ -42,17 +42,17 @@ Several, as listed below, are supported, and while logic to interface these prog
 Qiskit Nature via the above pip installation, the dependent programs/libraries themselves need
 to be installed separately.
 
-1. [Gaussian 16&trade;](https://qiskit.org/documentation/nature/apidocs/qiskit_nature.second_q.drivers.gaussiand.html), a commercial chemistry program
```

Comment:
The README was updated here and reflects the main branch code, as can be seen in the sample python code where `second_q` is imported, and the links here reflect that but since the next release has not been published yet nor have the docs corresponding to it they do not as yet function. The README on the stable branch, which is what is released is self-consistent the second_quantization way with links and sample code, so they work there.

Reply:
ok

Reply:
Steve is correct here. Nonetheless, could we update the pylint link so if you revert the changes to the links here in the README, we can still merge this PR

Reply:
Sure! Thanks for point it out.